src/lib/eslint/LWCEslintStrategy.ts

```diff
@@ -0,0 +1,79 @@
+import { EslintStrategy } from './BaseEslintEngine';
+import {ENGINE, LANGUAGE, LWC_RULE_PREFIX} from '../../Constants';
+import {RuleViolation} from '../../types';
+import { Logger } from '@salesforce/core';
+
+const ES_CONFIG = {
+	"parser": "babel-eslint",
+	"plugins": ["@lwc/eslint-plugin-lwc"],
+	"baseConfig": {
+		"extends": ["@salesforce/eslint-config-lwc/base"]
+	},
+	"ignorePatterns": [
+		"node_modules/!**"
+	],
+	"useEslintrc": false, // TODO derive from existing eslintrc if found and desired
+	"resolvePluginsRelativeTo": __dirname, // Use the plugins found in the sfdx scanner installation directory
+	"cwd": __dirname // Use the parser found in the sfdx scanner installation
+};
+
+export class LWCEslintStrategy implements EslintStrategy {
+	private static LANGUAGES = [LANGUAGE.JAVASCRIPT];
```

Comment:
Just to confirm - are we not supporting typescript-based LWC files because the config would need a different parser that works with typescript code?

Reply:
At this point I think it makes sense to only support JavaScript. That is all that is supported by the eslint lwc plugin, and LWC doesn't yet officially support TypeScript https://trailblazer.salesforce.com/ideaView?id=0873A000000CPrdQAG